x/stakeibc/keeper/icacallbacks_undelegate_test.go

```diff
@@ -119,7 +119,7 @@ func (s *KeeperTestSuite) SetupUndelegateCallback() UndelegateCallbackTestCase {
 		SplitDelegations:        []*types.SplitDelegation{&val1SplitDelegation, &val2SplitDelegation},
 		EpochUnbondingRecordIds: []uint64{epochNumber},
 	}
-	callbackArgsBz, err := s.App.StakeibcKeeper.MarshalUndelegateCallbackArgs(s.Ctx, callbackArgs)
+	callbackArgsBz, err := proto.Marshal(&callbackArgs)
```

Comment:
Could you explain why this can now be simplified and why we can do away with `MarshalUndelegateCallbackArgs`?

Reply:
all `MarshalUndelegateCallbackArgs` and `UnmarshalUndelegateCallbackArgs` are doing is calling proto.Marshal, proto.Unmarshal under the hood!

I think we should do away with all our custom marshal/unmarshal functions cause they add ~20 lines of unnecessary code per callback and add an extra step/ more friction to the process of adding a new callback.